default.json

```diff
@@ -103,6 +103,15 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the README.md on renovatebot/github-action repository",
+      "fileMatch": ["README.md"],
+      "matchStrings": [
+        "insert regex that matches actions/checkout@v2.3.4 and matches renovatebot/github-action@v25.51.5"
+      ],
+      "datasourceTemplate": "docker",
+      "versioningTemplate": "docker"
```

Comment:
I don't know if GitHub Action releases are captured by this?

```
      "datasourceTemplate": "docker",
      "versioningTemplate": "docker"
```

Reply:
```suggestion
      "datasourceTemplate": "github-tags",
      "versioningTemplate": "semver"
```